Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN yum install epel-release -y && \
     yum remove epel-release -y && \
     yum clean all
 
-RUN pip install raven --upgrade && pip install django==1.11.2 psycopg2==2.7.3
+RUN pip install raven --upgrade && pip install django==1.11.2 psycopg2==2.7.3 \
+ djangorestframework drfdocs django-filter
 
```

Comment:
why we are adding djangorestframe worker to workers? we are running the data retrieval from the apiserver container right?